actions/entrypoint.sh

```diff
@@ -0,0 +1,4 @@
+#! /usr/bin/env bash
+
+args=("$@")
+ggshield secret scan -v ${args[@]} ci
```

Comment:
I don't know the syntax used here for `args`. What does it bring compared to the simpler `ggshield secret scan -v "$@" ci`?

Reply:
Sure, will update the file hierarchy if it doesn't make sense to have secret above iac. Unsure about the syntax. This was the working syntax in ggshield-actions, I will check

Reply:
It doesn't seem to work without the args trick. The help command is displayed

Reply:
Strange, it works here. Another Bash mystery I guess.